Kitodo/src/main/java/org/kitodo/production/helper/Helper.java

```diff
@@ -263,9 +263,9 @@ public static void setMessage(String control, String message, String description
      */
     private static void setMessage(String control, String message, String description, boolean onlyInfo) {
         // Never forget: Strings are immutable
-        message = message.replaceAll("<", "&lt;");
+        message = Objects.toString(message).replaceAll("<", "&lt;");
```

Comment:
Does this change introduce any different or it is only some cosmetical change?

Reply:
If the message is `null`, "null" appears as the message text in the interface. Previously, the interface did not show anything, because a NullPointerException occurred here.